src/contentLoader.js

```diff
@@ -0,0 +1,47 @@
+const displayArea = document.querySelector('.result');
+
+const contentLoader = (response) => {
+  displayArea.innerHTML = '';
+
+  const cityInfo = document.createElement('div');
+  cityInfo.classList.add('city-info');
+
+  const location = document.createElement('h3');
+  location.innerText = response.name;
+
+  const temperature = document.createElement('h3');
+  temperature.classList.add('temp');
+  temperature.innerHTML = `${Math.floor(response.main.temp)}<span>&#xb0;</span>C`;
+
+  temperature.addEventListener('click', () => {
+    if (temperature.innerHTML === `${Math.floor(response.main.temp)}<span>°</span>C`) {
+      temperature.innerHTML = `${Math.floor(((response.main.temp) / 5) * 9) + 32}<span>&#xb0;</span>F`;
+    } else {
+      temperature.innerHTML = `${Math.floor(response.main.temp)}<span>&#xb0;</span>C`;
+    }
+  });
```

Comment:
Please make request for both temperature units instead of calculating like you do here ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/requests_requirement.png)